src/features/chains/utils.ts

```diff
@@ -13,7 +14,10 @@ export function getChainDisplayName(chain: ChainName, shortName = false) {
 
 export function isPermissionlessChain(chain: ChainName) {
   if (!chain) return true;
-  return getChainMetadata(chain).protocol === ProtocolType.Ethereum || !chainMetadata[chain];
+  return (
+    getChainMetadata(chain).protocol !== ProtocolType.Ethereum ||
+    !CoreChains.includes(chain as CoreChain)
+  );
 }
```

Comment:
what does "permissionless" mean in the UI? I think over time we should prune these distinctions
imo the registry is the right place to draw these boundaries

Reply:
Agreed. https://github.com/hyperlane-xyz/hyperlane-registry/issues/42 works towards that.